content/blog/posts/join-our-team.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Join Our Team
+date: 2021-07-06
+authors:
+  - Eva Ruth Moravec
+hero: https://res.cloudinary.com/texas-justice-initiative/image/upload/v1625582253/TJJ-icon-only_imqijs.png
+---
+TJI functions thanks to talented [volunteers](https://texasjusticeinitiative.org/about) who work collaboratively to build and maintain our website and data tools. Our team meets weekly – remotely and, when volunteers in Austin are comfortable doing so again, in-person – and keep in touch during the week on Slack and email. We are lucky to have consistently had amazing volunteers, and we are looking to add a few people to the mix. 
+
+Our volunteers have a variety of skillsets, backgrounds and levels of experience. Frequently, we learn from and teach each other during our weekly meetings and in one-off conversations. We'd love to welcome folks who can work on existing projects like automating our data collection processes, incorporating maps into our existing data sets and applying our new style guide to our use of colors in existing charts, and we're also looking for people to fill these new roles: 
+
+* data visualization designer/architect for new  project that incorporates several sets of data regarding populations in county jails over time; 
+* data architect to coordinate with corporate volunteers on automation tool; 
+* SEO and Google Analytics whiz  
```

Comment:
There is an issue right now with our core stylesheet causing all lists not to get their basic styles. I'm hestitant to make a quick change which could break other parts of the site. Can you add a dash in front of the list items for now, until I can work on a fix and check it on each page?

```suggestion
* - data visualization designer/architect for new  project that incorporates several sets of data regarding populations in county jails over time; 
* - data architect to coordinate with corporate volunteers on automation tool; 
* - SEO and Google Analytics whiz  
```